packages/devtools_app/lib/src/screens/network/network_request_inspector_views.dart

```diff
@@ -421,6 +422,7 @@ class ImageResponseView extends StatelessWidget {
               prettyPrintBytes(
                 encodedResponse.lengthInBytes,
                 includeUnit: true,
+                kbFractionDigits: 0,
```

Comment:
doesn't really matter here as images are large but could probably be 1 for consistency.